File: CS/DataModel.Shared/BusinessObjects/Department.cs
using System;
using System.Collections.ObjectModel;
using DevExpress.Persistent.BaseImpl.EF;

namespace DataModel.Shared.BusinessObjects {
    public class Department : BaseObject {

        public virtual string Title { get; set; }

        public virtual string Office { get; set; }

        public virtual IList<Employee> Employees { get; set; } = new ObservableCollection<Employee>();

        public override string ToString() {
            return Title;
        }
    }
}

File: VB/DataModel.Shared/BusinessObjects/Department.vb
Imports System.Collections.ObjectModel
Imports DevExpress.Persistent.BaseImpl.EF

Namespace DataModel.Shared.BusinessObjects

    Public Class Department
        Inherits BaseObject

        Public Overridable Property Title As String

        Public Overridable Property Office As String

        Public Overridable Property Employees() As IList(Of Employee) = New ObservableCollection(Of Employee)()

        Public Overrides Function ToString() As String
            Return Title
        End Function
    End Class
End Namespace
